data/strategies/StrategyLenderYieldOptimiser.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "strategy",
```

Comment:
can we remove this in favor of this one: data/strategies/StrategyLenderYieldOptimiser - ca.json